### ISSUE TITLE
Product card creation errors

### ISSUE THREAD

**Issue by dominikahr** [allegro/allegro-api#9965]
Hi there, 

one of our users tried to create some product cards, but we got back some weird error messages. I tried to test it for the same category and with the same parameters on polish sandbox, but everything worked just fine. But on CZ production environment, it is not possible to create it. 

Trace id: 55df827ba7ddb660

(EAN 8586022596520)

![image](https://github.com/user-attachments/assets/f802843d-7733-4d32-b9e9-9cf3cc7b3dcb)

Everything in the request seems fine from our side. Can you please have a look at it and tell us what is wrong? 

Thanks in advance! 

All the best, 

Dominika

**Comment by MaciejFrackowiak** [comment]
You have to remember that category and parameters structure on production and Sandbox can be different because it is refreshed only once a quarter. 
The user tried to list an offer with a new product in the described case, where he provided ambiguous values for parameters Material and Dedicated brand (value - "other" - in Czech "ostatni"). The new product cannot have parameters with "other" values.

**Comment by dominikahr** [comment]
@MaciejFrackowiak we are always asking for up to date parameters and allowed values, so it is not possible, we do not have the newest data. 

Moreover I checked it on CZ Allegro via the listing form and for this category (Elektronika - Telefony a příslušenství - Příslušenství GSM - Obaly a kryty), it is possible to select "other" (ostatní) for these parameters:
![image](https://github.com/user-attachments/assets/f7698b67-9b8c-46ef-9290-dbe62b958c64)

**Comment by MaciejFrackowiak** [comment]
Yes, you can choose values, but you cannot create a new product with these values. The product cannot have a parameter with an "other" value.

**Comment by dominikahr** [comment]
@MaciejFrackowiak but it worked for us in the past months, that is what is confusing for us. Moreover for the parameters "Material" and "Dedicated brand" what should the seller do when none of the allowed values are correct for him? And why do you send "other" in the allowed values when it is not actually an allowed value?

**Comment by MaciejFrackowiak** [comment]
In such a case, the seller can contact Allegro support and ask to add new parameter values to the category and parameter tree. For every category, there are several identification rules, so for one category it can work with values "other", but for other no.
